README.md

```diff
@@ -18,4 +31,7 @@ $ make setup-integration
 $ make setup-e2e
 ```
 
+> **Warning**
```

Comment:
Add Option 3 for external etcd.

Reply:
I have added the `Option 3:  Run the multicluster-controlplane binary with external etcd` in README.md

In `Option 2` we use external etcd by default. So `Option 3` mainly describes the ability to specify an external etcd when starting a binary server. Starting a binary etcd is described in many [posts](https://medium.com/nirman-tech-blog/setting-up-etcd-cluster-with-tls-authentication-enabled-49c44e4151bb), so I won't go into detail.